label_studio/io_storages/gcs/utils.py

```diff
@@ -0,0 +1,231 @@
+import re
+import logging
+import google.cloud.storage as gcs
+import json
+import base64
+import google.auth
+
+from enum import Enum
+from urllib.parse import urlparse
+from datetime import timedelta
+from typing import Union
+from google.oauth2 import service_account
+from google.auth.exceptions import DefaultCredentialsError
+
+
+logger = logging.getLogger(__name__)
+
+Base64 = bytes
+
+
+class GCS(object):
+    _client_cache = {}
+    _credentials_cache = {}
+    DEFAULT_GOOGLE_PROJECT_ID = gcs.client._marker
+
+    class ConvertBlobTo(Enum):
+        NOTHING = 1
+        JSON = 2
+        JSON_DICT = 3
+        BASE64 = 4
+
+    @classmethod
+    def get_client(
+        cls,
+        google_project_id: str = None,
+        google_application_credentials: Union[str, dict] = None
+    ) -> gcs.Client:
+        """
```

Comment:
just curious: do we prefer Google-style docstrings? I've been using numpy style, fwiw

Reply:
@pakelley this was already merged in another PR